web/frontend/src/components/base/index.svelte

```diff
@@ -287,6 +287,12 @@
     angle = event.detail.value;
   };
 
+  const handleOnKeyUp = (event: KeyboardInput, callback: () => void) => {
+    if (event.KeyCode === 13) {
```

Comment:
`KeyCode` is deprecated, we should be using `code`: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode